com.unity.render-pipelines.universal/Shaders/Nature/SpeedTree8Passes.hlsl

```diff
@@ -91,6 +91,15 @@ struct SpeedTreeFragmentInput
 #endif
 };
 
+#define UNITY_INV_PI        0.31830988618f
+inline float GGXTerm (float NdotH, float roughness)
+{
+    float a2 = roughness * roughness;
+    float d = (NdotH * a2 - NdotH) * NdotH + 1.0f; // 2 mad
+    return UNITY_INV_PI * a2 / (d * d + 1e-7f); // This function is not intended to be running on Mobile,
+                                            // therefore epsilon is smaller than what can be represented by half
+}
+
```

Comment:
In BSDF.hlsl we seem to have the same functions for this calculations. In order not to duplicate things, I would recommend trying to use those.
Check the `real D_GGXNoPI(real NdotH, real roughness)` in that file.

Reply:
Changed in d2a779d